lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html

```diff
@@ -1,70 +1,61 @@
 <div [ngSwitch]="templateType">
     <div *ngSwitchDefault>
         <mat-form-field class="adf-property-field adf-card-textitem-field"
-            [ngClass]="{ 'adf-property-read-only': !isEditable, 'adf-property-field-has-error mat-form-field-invalid': isEditable && hasErrors }"
-            [floatLabel]="'never'"
+            [ngClass]="{
+                'adf-property-read-only': !isEditable
+            }"
+            [floatLabel]="'always'"
             appearance="standard">
+
             <mat-label *ngIf="showProperty || isEditable" [attr.data-automation-id]="'card-textitem-label-' + property.key" class="adf-property-label">
                 {{ property.label | translate }}
             </mat-label>
-                <input matInput
-                    *ngIf="!property.multiline"
-                    class="adf-property-value"
-                    title="{{property.label | translate }}"
-                    [placeholder]="property.default"
-                    [attr.aria-label]="property.label | translate"
-                    [formControl]="textInput"
-                    (dblclick)="copyToClipboard(property.displayValue)"
-                    matTooltipShowDelay="1000"
-                    [matTooltip]="'CORE.METADATA.ACTIONS.COPY_TO_CLIPBOARD' | translate"
-                    [matTooltipDisabled]="isEditable"
-                    [attr.data-automation-id]="'card-textitem-value-' + property.key"
-                    (keydown)="undoText($event)">
-                <textarea matInput
-                    *ngIf="property.multiline"
-                    title="{{property.label | translate }}"
-                    [cdkTextareaAutosize]="true"
-                    [cdkAutosizeMaxRows]="1"
-                    [cdkAutosizeMaxRows]="5"
-                    class="adf-property-value"
-                    [placeholder]="property.default"
-                    [attr.aria-label]="property.label | translate"
-                    [formControl]="textInput"
-                    [attr.data-automation-id]="'card-textitem-value-' + property.key">
-                </textarea>
-
-                <button
-                    *ngIf="isEditable"
-                    matSuffix
-                    class="adf-textitem-clear-icon"
-                    [attr.aria-label]="'CORE.METADATA.ACTIONS.CLEAR' | translate"
-                    (click)="clearValue()">
-                    <mat-icon>cancel</mat-icon>
-                </button>
-                <button
-                    *ngIf="isEditable"
-                    matSuffix
-                    class="adf-textitem-edit-icon"
-                    [attr.title]="'CORE.METADATA.ACTIONS.EDIT' | translate">
-                    <mat-icon>mode_edit</mat-icon>
-                </button>
 
+            <input matInput
+                *ngIf="!property.multiline"
+                class="adf-property-value"
+                [ngClass]="{
+                    'adf-property-value-editable': editable,
```

Comment:
I don't see this class in scss file for this component. Is not that missing?

Reply:
the class is in the card view for shared use